osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/alarm/ReadAlarmRegisterCommandExecutor.java

```diff
@@ -37,13 +38,16 @@ public class ReadAlarmRegisterCommandExecutor
   private static final Logger LOGGER =
       LoggerFactory.getLogger(ReadAlarmRegisterCommandExecutor.class);
 
-  final DlmsObjectConfigService dlmsObjectConfigService;
+  final ObjectConfigServiceHelper objectConfigServiceHelper;
+  static final int ALARM_REGISTER_1_ATTRIBUTE_ID = 2;
```

Comment:
Alarm register 1 and 2 are both of type DLMS class Register. So the attribute id is the same for both. To avoid the magic number, you could use RegisterAttribute.VALUE.attributeId().

Reply:
The attribute id value is updated to RegisterAttribute.VALUE.attributeId().